### PR TITLE
Add July 2024 newsletter

### PR DESCRIPTION
**Pull request by ericholscher**
Fixes https://github.com/readthedocs/website/issues/306

Please add any additional items in there as GH comments, or feel free to push them up to the PR itself. :+1:

----
📚 Documentation preview 📚: https://readthedocs-about--309.org.readthedocs.build/

### CODE REVIEW

File: content/posts/newsletter-july-2024.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+category: Newsletter
+tags: newsletter, community
+authors: Eric Holscher
+status: published
+image: /images/july-2024.jpg
+image_credit: Photo by <a href="https://unsplash.com/@dyc251?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Diana Yohannes</a> on <a href="https://unsplash.com/photos/a-tree-in-front-of-a-sunset-vmNdWNX3xpU?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+We missed a couple of newsletter updates because of travel and other commitments, so we're back with a combined update for the last couple months.
+
+## News and updates
```
Comment (humitos) [inline content/posts/newsletter-july-2024.md:13]: We could mention that `build.commands` is not in beta anymore: https://github.com/readthedocs/readthedocs.org/pull/11411

Comment (humitos) [inline content/posts/newsletter-july-2024.md:13]: We also support Ubuntu 24.04 for builds now.

Comment (humitos) [inline content/posts/newsletter-july-2024.md:13]: All the `build.tools` were updated to their latest versions: https://github.com/readthedocs/readthedocs.org/pull/11386

Comment (humitos) [inline content/posts/newsletter-july-2024.md:13]: We've deprecated support for all the VCS except Git.

**Review by humitos — COMMENTED** [review]
I added a few things that happened this month as well.

**Review by agjohnson — APPROVED** [review]
Post looks good, but I agree on a few of the additions mentioned too. I don't have anything to add to the list otherwise.

**Comment by ericholscher** [comment]
I'll plan to merge this tomorrow 👍

**Pull request merged**